test/minikube/minikube_upgrade_helm_test.go

```diff
@@ -111,47 +106,48 @@ func upgradeDatabaseTest(t *testing.T, fromHelmVersion string, updateOptions *Up
 
 	helm.Upgrade(t, options, testlib.ADMIN_HELM_CHART_PATH, helmChartReleaseName)
 
-	if updateOptions.adminPodShouldGetRecreated {
+	if upgradeOptions.AdminPodShouldGetRecreated {
 		testlib.AwaitPodObjectRecreated(t, namespaceName, adminPod, 30*time.Second)
 	}
 
-	testlib.AwaitPodUp(t, namespaceName, admin0, 300*time.Second)
+	testlib.AwaitPodUp(t, namespaceName, admin0, 600*time.Second)
```

Comment:
The 300s default is there in case the k8s node needs to pull the docker image and the network is slow. What is the admin doing for 10 minutes? Is it crash looping? is it a product problem? Environmental?

Reply:
Yeah, this change was bogus and I backed it out in my latest commit. It was the timeout on the database check that I wanted to increase.